src/Mod/ReverseEngineering/App/AppReverseEngineering.cpp

```diff
@@ -999,7 +999,7 @@ PyMOD_INIT_FUNC(ReverseEngineering)
     }
 
     PyObject* mod = Reen::initModule();
-    Base::Console().log("Loading ReverseEngineering module... done\n");
+    Base::Console().log("Loading ReverseEngineering module… done\n");

```

Comment:
```suggestion
    Base::Console().log("Loading Reverse Engineering module… done\n");
```
The workbench actually has a space in its name